macro/run_simScript.py

```diff
@@ -138,6 +138,7 @@
 parser.add_argument("--scName", help="The name of the SC shield in the database", default="sc_v6")
 parser.add_argument("--MesonMother",   dest="MM",  help="Choose DP production meson source", required=False,  default=True)
 parser.add_argument("--debug",  help="1: print weights and field 2: make overlap check", required=False, default=0, type=int, choices=range(0,3))
+parser.add_argument("--helium",  help="use helium bag for vacuum tank", required=False, action="store_const", const="helium", default="vacuums")
```

Comment:
The `required=False` is redundant here.

Nice solution with the `store_const`

I would rephrase the help message a bit, as it might be misunderstood as replacing the tank structure as well (and break the long line), but this is really nitpicking.